tor.mk

```diff
@@ -30,13 +34,12 @@ endif
 tor-package: tor-stage
 	# tor.mk Package Structure
 	rm -rf $(BUILD_DIST)/tor
-	mkdir -p $(BUILD_DIST)/tor
+	mkdir -p $(BUILD_DIST)/tor/etc
 	mkdir -p $(BUILD_DIST)/tor/usr
```

Comment:
You can get rid of all of these and just `cp -a $(BUILD_STAGE)/tor $(BUILD_DIST)`